virtcontainers/container.go

```diff
@@ -785,7 +785,7 @@ func newContainer(sandbox *Sandbox, contConfig ContainerConfig) (*Container, err
 		return nil, err
 	}
 
-	if err = c.createDevices(contConfig); err != nil {
+	if err = c.createDevices(*contConfig); err != nil {
```

Comment:
For c.createDevice function,  I think you can also changed it's parameter from instance to pointer.

Reply:
Yes, we can change it in code. But I don't think we can change its semantics, we cannot change contConfig at **create**Devices procedure.

Reply:
What I mean is change it in code, since there is no need to pass an config instance here.

Reply:
OK, I'll change it.
Should I change and force-push all this two commits again, or just add a new commit?
which one is better?

Reply:
It's better to change them in a single commit and do a force push